### PR TITLE
Add changelog  for release v0.20.0

### PR DESCRIPTION
**Pull request by rahulbabu95**
*Description of changes:*
Add change log for release v0.20.0

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -24,13 +24,55 @@ description: >
...
+  - [`v1-30-eks-7`](https://distro.eks.amazonaws.com/releases/1-30/7/)
+- Cilium: `v1.13.9` to `v1.13.16`
+- Cluster API: `v1.6.1` to `v1.7.2`
+- Cluster API Provider vSphere: `v1.8.5` to `v1.10.0`
+- Cluster API Provider Nutanix: `v1.2.3` to `v1.3.1`
+- Flux: `v2.2.3` to `v2.3.0`
+- Kube-vip: `v0.7.0` to `v0.8.0`
+- Image-builder: `v0.1.24` to `v0.1.26`
+- Kind: `v0.22.0` to `v0.23.0`
+- Etcdadm Controller: `v1.0.17` to `v1.0.21`
```
Comment (rahulbabu95) [inline docs/content/en/docs/whatsnew/changelog.md:67]: should tink stack changes be included here.?

File: docs/content/en/docs/whatsnew/changelog.md [hunk 2]
```diff
@@ -24,13 +24,55 @@ description: >
...
+| Bottlerocket 1.20.0 |    ✔    |     -      |    —    |     —      |  —   |
+| Bottlerocket 1.19.4 |    -    |     \*      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     ✔      |    ✔    |     ✔      |  —   |
+
+\* [EKS Anywhere issue regarding deprecation of Bottlerocket bare metal variants](https://github.com/aws/eks-anywhere/issues/7754)
+
+### Added
+- Support for Kubernetes v1.30
+- Support for configuring api server flags in cluster spec ([#7755](https://github.com/aws/eks-anywhere/pull/7755))
```
Comment (abhinavmpandey08) [inline docs/content/en/docs/whatsnew/changelog.md:49]: nit
```suggestion
- Support for configuring kube-apiserver flags in cluster spec ([#7755](https://github.com/aws/eks-anywhere/pull/7755))
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 3]
```diff
@@ -24,13 +24,55 @@ description: >
...
+- Support for control plane failure domains on Nutanix ([#8192](https://github.com/aws/eks-anywhere/pull/8192))
+
+### Changed
+- Generate cluster config command includes OSImageURL in tinkerbell machine config objects ([#8226](https://github.com/aws/eks-anywhere/pull/8226))
+- Added EKS-D for 1-30:
+  - [`v1-30-eks-7`](https://distro.eks.amazonaws.com/releases/1-30/7/)
+- Cilium: `v1.13.9` to `v1.13.16`
+- Cluster API: `v1.6.1` to `v1.7.2`
+- Cluster API Provider vSphere: `v1.8.5` to `v1.10.0`
+- Cluster API Provider Nutanix: `v1.2.3` to `v1.3.1`
```
Comment (abhinavmpandey08) [inline docs/content/en/docs/whatsnew/changelog.md:62]: https://github.com/aws/eks-anywhere-build-tooling/blob/release-0.20/projects/nutanix-cloud-native/cluster-api-provider-nutanix/GIT_TAG
```suggestion
- Cluster API Provider Nutanix: `v1.2.3` to `v1.3.5`
```

File: docs/content/en/docs/whatsnew/changelog.md [hunk 4]
```diff
@@ -24,13 +24,55 @@ description: >
...
+|      RHEL 9.x       |    —    |     ✔      |    ✔    |     ✔      |  —   |
+
+\* [EKS Anywhere issue regarding deprecation of Bottlerocket bare metal variants](https://github.com/aws/eks-anywhere/issues/7754)
+
+### Added
+- Support for Kubernetes v1.30
+- Support for configuring api server flags in cluster spec ([#7755](https://github.com/aws/eks-anywhere/pull/7755))
+- Redhat 9 support for Bare Metal ([#3032](https://github.com/aws/eks-anywhere-build-tooling/pull/3032))
+- Support for configuring kubelet settings in cluster spec ([#8265](https://github.com/aws/eks-anywhere/pull/8265))
+- Support for kubelet settings on BottleRocket nodes ([#3266](https://github.com/aws/eks-anywhere-build-tooling/pull/3266))
```
[the remark this answers is not in the document]
Comment (mitalipaygude) [reply docs/content/en/docs/whatsnew/changelog.md:52]: Yeah I agree, we don't need to call it out differently.

File: docs/content/en/docs/whatsnew/changelog.md [hunk 5]
```diff
@@ -24,13 +24,55 @@ description: >
...
+- Kind: `v0.22.0` to `v0.23.0`
+- Etcdadm Controller: `v1.0.17` to `v1.0.21`
+
+### Removed
+- Support for Kubernetes v1.25
+- Support for certain curated packages CLI commands ([#8240](https://github.com/aws/eks-anywhere/pull/8240))
+
+### Fixed
+- CLI commands for packages to honor the registry mirror setup in cluster spec ([#8026](https://github.com/aws/eks-anywhere/pull/8026))
+
```
Comment (sp1999) [inline docs/content/en/docs/whatsnew/changelog.md:75]:
```suggestion
- Cluster status reconciliation for worker nodes when cluster autoscaler is configured ([8254](https://github.com/aws/eks-anywhere/pull/8254))

```

File: docs/content/en/docs/whatsnew/changelog.md (hunk 1 above)
Comment (ahreehong) [reply docs/content/en/docs/whatsnew/changelog.md:67]: - Tinkerbell stack (https://github.com/aws/eks-anywhere/issues/3233):
    Cluster API Provider Tinkerbell v0.4.0 -> v0.5.3
    Hegel v0.10.1 -> v0.12.0
    Rufio afd7cd82fa08dae8f9f3ffac96eb030176f3abbd -> v0.3.3
    Tink v0.8.0 -> v0.10.0
    Boots/Smee v0.8.1 -> v0.11.0
    Charts: v0.4.5

>**_NOTE_**: The Boots service has been renamed to Smee by the upstream tinkerbell community with this version upgrade. Any reference to Boots or Smee in our docs refer to the same service.

File: docs/content/en/docs/whatsnew/changelog.md (hunk 5 above)
Comment (rahulbabu95) [reply docs/content/en/docs/whatsnew/changelog.md:75]: this is already in [release-0.19](https://github.com/aws/eks-anywhere/pull/8283) ?

**Review by abhinavmpandey08 — APPROVED** [review]

**Comment by mitalipaygude** [comment]
Looks good to me!

**Pull request merged**